utils/releaser/src/FilesProvider/DockerComposeFilesProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\FilesProvider;
+
+use Symfony\Component\Finder\Finder;
+
+final class DockerComposeFilesProvider
+{
+    /**
+     * @return \Symfony\Component\Finder\SplFileInfo[]
+     */
+    public function provide(): array
+    {
+        $finder = Finder::create()
+            ->in(getcwd())
+            ->exclude('vendor')
```

Comment:
even here we need to add `->exclude('project-base/var')`, otherwise it fails on 
```
[Symfony\Component\Finder\Exception\AccessDeniedException]                                                                    
  RecursiveDirectoryIterator::__construct(/var/www/html/project-base/var/postgres-data): failed to open dir: Permission denied
```